CHANGELOG.md

```diff
@@ -43,6 +44,7 @@ should change the heading of the (upcoming) version to include a major version b
 ## Dev / docs / playground
 
 - Updated docs and playground with the implementation guide of newly added date re-order feature.
+- [4024](https://github.com/rjsf-team/react-jsonschema-form/issues/4024) Updated the base64 references from (`atob` and `btoa`) to invoke the functions from the new `base64` object in `@rjsf/utils`.
```

Comment:
Also, move this into a new `Dev / docs / playground` sub-section under `5.16.2`

Reply:
fixed